rust/projects/file-io/project.md

```diff
@@ -145,24 +180,33 @@ disk.
 
 [`serde`]: https://serde.rs/
 
-This is the basic behavior of `kvs` with a write-ahead log:
+This is the basic behavior of `kvs` with a log:
 
 - "set"
   - The user invokes `kvs set mykey, myvalue`
   - `kvs` creates a struct representing the "get" command, containing its key and
     value
   - It then serializes that command to a `String`
-  - It then appends the serialized command to a file containing the WAL
+  - It then appends the serialized command to a file containing the log
   - If that succeeds, it exits silently with error code 0
   - If it fails, it exits by printing the error and return a non-zero error code
 - "get"
```

Comment:
Do we really need to state how "get" works here in "Part 2: Storing writes in the log"?

Reply:
Shall we move the more general behavior of the two subcommands to "Project spec" and leave the details in these two parts?

Reply:
Perhaps not. I was thinking that describing "get" would give context to the "set" algorithm, and since it's described here it is _not_ described in the section implementing get.

Reply:
For the moment I've left it per my explanation. Feel free to object.